lib/kvdb/kvdb_rparams.c

```diff
@@ -445,6 +620,62 @@ throttle_init_policy_converter(
     return true;
 }
 
+static merr_t
+throttle_init_policy_stringify(
+    const struct param_spec *const ps,
+    const void *const              value,
+    char *const                    buf,
+    const size_t                   buf_sz,
+    size_t *const                  needed_sz)
+{
+    size_t      n;
+    const char *param;
+
+    INVARIANT(ps);
+    INVARIANT(value);
+    INVARIANT(buf);
+
+    switch (*(uint *)value) {
+        case THROTTLE_DELAY_START_DEFAULT:
+            param = "\"default\"";
+            break;
+        case THROTTLE_DELAY_START_LIGHT:
+            param = "\"light\"";
+            break;
+        case THROTTLE_DELAY_START_MEDIUM:
+            param = "\"medium\"";
+            break;
+        default:
+            assert(false);
```

Comment:
return `merr(EINVAL)`.  I presume you'll make that change in all these default cases?

Reply:
I think I am just going to make this an abort() instead. Getting to this case is some sort inconsistency that should never-ever happen.

Reply:
Is there any way for this to bring down a running system?  A poorly formatted REST request, for example?

Reply:
No. You can't be given invalid data due to the checks within the conversion and validation functions.